sycl/test/usm/depends_on.cpp

```diff
@@ -39,7 +39,9 @@ int main() {
     return -1;
   }
 
+  event e;
   auto eInit = q.submit([&](handler &cgh) {
+    cgh.depends_on(e);
```

Comment:
What should happen here?
`e` is not initialized.
Looks like the behavior of this test now is undefined.

Reply:
Constructor is run: 
From spec:
Constructs a ready SYCL event. If the constructed SYCL event is waited on it will complete immediately
--

Reply:
event e;
e.wait();

This is legal and should immediately return as Robert mentions (Section 4.6.6).

Reply:
Okay. Thanks for clarification.